### DISCUSSION TITLE
Add user friendly status to vacuum entity

### DISCUSSION THREAD

**Discussion by simontegelid** [Thomas55555/husqvarna_automower#91] [Ideas]
Hi,

I'm using the lovelace vacuum card (https://github.com/denysdovhan/vacuum-card) for my automower and the missing status text (ATTR_STATUS) makes it pretty rough to look at. I'm willing to add a ATTR_STATUS to the custom attributes containing the same strings as in the native Husqvarna app to make it look a bit nicer. An example is from the Neato integration (https://github.com/home-assistant/core/blob/e5d6e18e30ba8602bdeec02d665a58dfc6798aed/homeassistant/components/neato/vacuum.py#L182) where the ATTR_STATUS is used. Would such a PR get merged?

For example, in the following screenshot, "RESTRICTED" would be replaced with "Parked":

![image](https://user-images.githubusercontent.com/891850/130403481-ebb4e856-2aaa-4b23-9fe1-e79f5b2efd05.png)

**Comment by Thomas55555** [comment]
Hi,
it was a little bit like that before #46. I had in the state of the mower a readable text, like in the app. But then some functions where not working. Like the clean button. So I decided to remove the status and only work with the standard vacuum states. But as I see, in the neato 
integration it's working.
How is it looking in the UI, is there just an extra line in the attributes with the status?
Then I think it's an good idea and we can merge it.
![DeepinBildschirmfoto_Bereich auswählen_20210823143550](https://user-images.githubusercontent.com/59625598/130448158-37b4de86-ba65-4581-aad8-c08d84021e5e.png)

**Reply by simontegelid** [reply]
The status and state are two different things, that's why it's working in the neato integration :) See #93
